answerking_app/tests/test_items.py

```diff
@@ -346,23 +430,15 @@ def test_put_valid_returns_ok(self):
         self.assertEqual(expected, actual)
         self.assertEqual(response.status_code, 200)
 
-    def test_put_invalid_id_returns_bad_request(self):
-        # Arrange
```

Comment:
why was this test deleted?

Reply:
Because it's actually changed and returns not found now I can add it back but there is not point, it's already tested on delete and get.
This test changes can be moved in a separate PR